StudioCore/Scene/RenderableProxy.cs

```diff
@@ -670,6 +672,7 @@ public void OnProviderAvailable()
                         }
                     }
                 }
+                MeshIndexCount = _meshProvider.IndexCount; //Used for model markers for old Navmeshes. Is this the best place for this check??
```

Comment:
This doesn't seem like the right place. Is this needed for navmeshes to render?

Reply:
Yes, for DS1- navmesh rendering. Model marker system checks mesh's submesh count, which DS1 navmeshes don't use which causes rendering to collapse (and I failed to figure out why exactly rendering failed because of this, but I think disposing meshes was going wrong somewhere - a pre-me problem)

Checking index count seemed like a pretty stupid hot-glue solution (even to me) but in the interest of maintaining the (poorly implemented, but incredibly useful) model marker system while restoring navmesh rendering functionality.

So, a simple change to make to make things work in lieu of the inevitable model marker system rewrite that somebody smarter than me would have to do.